content/codespaces/getting-started-with-codespaces/getting-started-with-your-python-project-in-codespaces.md

```diff
@@ -0,0 +1,221 @@
+product: '{% data reusables.gated-features.codespaces %}'
+versions:
+  free-pro-team: '*'
+topics:
+  - Codespaces
+---
+
+
+### Introduction
+
+This guide shows you how to set up your Python project in {% data variables.product.prodname_codespaces %}. It will take you through an example of opening your project in a codespace, and adding and modifying a dev container configuration from a template.
+
+#### Prerequisites 
+
+- You should have an existing Python project in a repository on {% data variables.product.prodname_dotcom_the_website %}. If you don't have a project, you can try this tutorial with the following example: https://github.com/2percentsilk/python-docs-hello-world.
+- You must have {% data variables.product.prodname_codespaces %} enabled for your organization.
+
+### Step 1: Open your project in a codespace
+
+1. Navigate to your project's repository. Use the {% octicon "download" aria-label="The download icon" %} **Code** drop-down menu, and select **Open with Codespaces**. If you don’t see this option, your project isn’t available for {% data variables.product.prodname_codespaces %}.
+  
+  ![Open with Codespaces button](/assets/images/help/codespaces/open-with-codespaces-button.png) 
+
+2. To create a new codespace, click {% octicon "plus" aria-label="The plus icon" %} **New codespace**.
+  ![New codespace button](/assets/images/help/codespaces/new-codespace-button.png)
+
+When you create a codespace, your project is created on a remote VM that is dedicated to you. By default, the container for your codespace has many languages and runtimes including Node.js, JavaScript, Typescript, nvm, npm, and yarn. It also includes a common set of tools like git, wget, rsync, openssh, and nano. 
+
+You can customize your codespace by adjusting the amount of vCPUs and RAM], [adding dotfiles to personalize your environment](/codespaces/setting-up-your-codespace/personalizing-codespaces-for-your-account), or by modifying the tools and scripts installed. 
+
+{% data variables.product.prodname_codespaces %} uses a file called `devcontainer.json` to store configurations. On launch {% data variables.product.prodname_codespaces %} uses the file to install any tools, dependencies, or other set up that might be needed for the project. For more information, see "[Configuring Codespaces for your project](/codespaces/setting-up-your-codespace/configuring-codespaces-for-your-project)."
+
+The next section shows you how to modify your tools by adding a dev container.
+
+### Step 2: Add a dev container to your codespace from a template  
+
+The default codespaces container will support running Python projects like [python-docs-hello-world](https://github.com/2percentsilk/python-docs-hello-world) out of the box. By setting up a custom container you can customize the tools and scripts that run as part of codespace creation and ensure a fully reproducible environment for all {% data variables.product.prodname_codespaces %} users in your repository.
```

Comment:
This doesn't really provide information about the types of projects it'll support out of the box. What specific features/options will/won't work in the default container? Will it run Jupyter? Anaconda? Common Python libraries?

Reply:
Good point - I can add some more clarity here!